proposals/vip_12.ts

```diff
@@ -0,0 +1,17 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const vip_x: ProposalDescription = {
+  title: 'VIP-12: Add new PSMs to PCV Guardian Whitelist',
+  commands: [
+    {
+      target: 'pcvGuardian',
+      values: '0',
+      method: 'addWhitelistAddresses(address[])',
+      arguments: [['{voltV2DaiPriceBoundPSM}', '{voltV2UsdcPriceBoundPSM}']],
+      description: 'Add new DAI, and USDC PSMs to PCV Guardian whitelist'
+    }
```

Comment:
This proposal should also pull all funds from the old PSM's so that the migrator or migrator router contract must be used.